extensions/positron-python/python_files/posit/positron/positron_jedilsp.py

```diff
@@ -285,10 +286,16 @@ def lsp_initialize(self, params: InitializeParams) -> InitializeResult:
             server.show_message_log(msg, msg_type=MessageType.Error)
             initialization_options = PositronInitializationOptions()
 
-        # If a notebook path was provided, set the project path to the notebook's parent.
-        # See https://github.com/posit-dev/positron/issues/5948.
+        # Set the project path to the notebook's parent if `working_directory` is not provided.
```

Comment:
What do you think about removing `notebook_path` and handling that case via `working_directory` as well?